tests/forcetorque/ForceTorqueTest.cc

```diff
@@ -0,0 +1,44 @@
+#include <gtest/gtest.h>
+#include <yarp/os/Bottle.h>
+#include <yarp/os/BufferedPort.h>
+#include <yarp/os/LogStream.h>
+#include <yarp/os/Network.h>
+#include <gz/sim/TestFixture.hh>
+
+TEST(ForceTorqueTest, PluginTest)
+{
+  yarp::os::NetworkBase::setLocalMode(true);
+  
+  // Maximum verbosity helps with debugging
+  gz::common::Console::SetVerbosity(4);
+
+  // Instantiate test fixture
+  gz::sim::TestFixture fixture("../../../tests/forcetorque/model.sdf");
+
+  int iterations = 1000;
+  fixture.Server()->Run(true, iterations, false);
```

Comment:
This is a bit difficult to read, see https://abseil.io/tips/94 for a description way.
```suggestion
  fixture.Server()->Run(/*_blocking=*/true, iterations, /*_paused=*/false);
```